pypesto/predict/amici_predictor.py

```diff
@@ -46,36 +46,37 @@ def __init__(self,
             be available for the post-processors
         post_processor:
             A callable function which applies postprocessing to the simulation
-            results and possibly defines different observables than those of
-            the amici model. Default are the observables of the amici model.
-            This method takes a list of dicts (with the returned fields ['t'],
-            ['x'], and ['y'] of the amici ReturnData objects) as input.
-            Safeguards for, e.g., failure of Amici are left to the user.
+            results and possibly defines different outputs than those of
+            the amici model. Default are the observables (`AMICI_Y`) of the
+            AMICI model. This method takes a list of dicts (with the returned
+            fields `AMICI_T`, `AMICI_X`, and `AMICI_Y` of the AMICI ReturnData
+            objects) as input. Safeguards for, e.g., failure of AMICI are left
+            to the user.
```

Comment:
Clarify that e.g. `AMICI_Y` refers to the constant name as defined in pypesto.predict.constants? This threw me off on first reading.